examples/Peripherals/HardwareTimer/countermode_up_down_centered/countermode_up_down_centered.ino

```diff
@@ -0,0 +1,123 @@
+/*
+  Countermode up down centered
+  This example shows how to configure HardwareTimer's counter mode.
+  
+  The following countermodes are demonstrated:
+  LL_TIM_COUNTERMODE_UP
+   _ — ‾ | _ — ‾ | _ — ‾ | _ — ‾ |
+  
+  LL_TIM_COUNTERMODE_DOWN
+   ‾ — _ | ‾ — _ | ‾ — _ | ‾ — _ |
+  
+  LL_TIM_COUNTERMODE_CENTER_UP_DOWN (with over/undeflow callback)
+  _ — ‾ | ‾ — _ | _ — ‾ | ‾ — _ |
+
+  LL_TIM_COUNTERMODE_CENTER_UP_DOWN (with only undeflow callback)
+  _ — ‾ ‾ — _ | _ — ‾ ‾ — _ |
+*/
+
+/*
+
+*/
+
+#if !defined(STM32_CORE_VERSION) || (STM32_CORE_VERSION < 0x01090000)
+#error "Due to API change, this sketch is compatible with STM32_CORE_VERSION  >= 0x01090000"
+#endif
+
+// Change Serial1 pins or use Serial (if available)
+#define TX PB3
+#define RX PB4
+HardwareSerial Serial1(RX, TX);
```

Comment:
To be more generic, I would have keep Serial as default (and eventually put in comment the changes for Serial1)

Reply:
Agree